understanding/20/use-of-color.html

```diff
@@ -81,7 +81,29 @@ <h2>Intent of Use of Color</h2>
          </p>
 
       </div>
-      
+
+      <div class="note">
+
+         <p>Under certain conditions, most user agents provide users with a color-only cue that a link has been activated by them ("visited"). However, several technical constraints result in authors having very limited control over these color-only indications of visited links.  The technical constraints are as follows:
```

Comment:
`a color-only cue that a link has been activated`

re-reading this, wonder if this may be dangerously close to the idea of `:active`? maybe better to be even more explicit, a la "...that a links points to a resource that has previously been visited"

Reply:
> that a links points to a resource that has previously been visited

But is that really what visited link state indicates? Isn't it more accurate to say that it represents that a link has been previously activated? After all,  the target might have timed out -- or the target may even have been modified. Doesn't the link get changed to 'visited' when it has been activated, not when the target is visited? In other words, there is no auditing of a successful connection AFAIK.

Reply:
well then, perhaps "previously activated"? just something to avoid confounding matters with the actual momentary `:active` state...